### PR TITLE
Fixes issue with images not displaying in IE10 due to window.location.origin not being defined in IE10

### PR DESCRIPTION
**Pull request by rgarmsen2295**
Meant to go in for 0.6.0.

### CODE REVIEW

**Comment by crspeller** [comment]
Ugggg. No window.location.origin in IE. Can we make this a function in utils so we don't have copy code everywhere for this?

**Comment by rgarmsen2295** [comment]
Yeah I know, IE is the gift the keeps on giving :P and yeah let me do that, should have done it in the first place.

**Comment by rgarmsen2295** [comment]
Ok cleaned it up, please take another look when you get a chance, thanks!

**Comment by crspeller** [comment]
I searched for window.location.origin in the code and there are a few more places where it is used. Can you convert them all to use the new method?

**Comment by rgarmsen2295** [comment]
Yup, will do, surprised these didn't cause noticeable issues earlier.

**Comment by rgarmsen2295** [comment]
Okay changed it to the new method in the few other places it was hanging around, let me know if you see anything else awry.

**Comment by crspeller** [comment]
Looks good to me.

File: web/react/stores/team_store.jsx [hunk 1]
```diff
@@ -11,6 +11,12 @@ var BrowserStore = require('../stores/browser_store.jsx');
 
 var CHANGE_EVENT = 'change';
 
+var utils;
+function getWindowLocationOrigin() {
+    if (!utils) utils = require('../utils/utils.jsx');
```
Comment (jwilander) [inline web/react/stores/team_store.jsx]: Why not just include ```utils = require('../utils/utils.jsx');``` with the rest of the requires? It's likely going to be needed later in other functions so you don't want to have to repeat this code everywhere

Comment (rgarmsen2295) [inline web/react/stores/team_store.jsx]: Ah, so this threw me for a loop for awhile (still not sure why exactly, maybe you can help there), but every time I tried to do var utils = ... the site would not work due to some random function in user store not being defined.  Once I realized that it was the including of the utils file that was causing it, I started looking at the other stores files to see if utils was defined anywhere else in a different one.  It wasn't _but_ I noticed that in browser_store.jsx we don't define UserStore until it's needed and figured I'd try that approach.  Once that worked I assumed it was something that was needed due to some ordering issue with how we load in the files.  But like you said not a great looking solution so let me know if you can think of something.

**Comment by coreyhulen** [comment]
Is this good to go?

**Comment by crspeller** [comment]
I was waiting for the @jwilander @rgarmsen2295 conversation above to reach a conclusion.

File: web/react/stores/team_store.jsx (hunk 1 above)
Comment (jwilander) [inline web/react/stores/team_store.jsx]: Ok, that's fine then

**Comment by jwilander** [comment]
Yup, good to go

**Pull request merged**